cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -463,6 +463,39 @@ def test_snapshot_history_btn(create_vm, provider):
     assert snapshot_view.is_displayed
 
 
+def test_snapshot_link_after_delete(create_vm, provider):
```

Comment:
If this is automating `test_snapshot_link_after_deleting_snapshot`, then can you delete that test too? Also add the appropriate tier and meta markers and Bugzilla docstring info.

Reply:
Agreed. Done.